src/classes/MyOtherClass.cls

```diff
@@ -0,0 +1,23 @@
+
+
+
+
+
+
+
+public class MyOtherClass {
```

Comment:
Class `MyOtherClass` does not enforce sharing correctly. Please change to ${suggestion}.